cmd/kube-apiserver/app/server.go

```diff
@@ -581,8 +582,9 @@ func BuildStorageFactory(s *options.ServerRunOptions) (*serverstorage.DefaultSto
 		storageFactory.SetEtcdLocation(groupResource, servers)
 	}
 
-	if s.Etcd.EncryptionProviderConfigFilepath != "" {
-		transformerOverrides, err := encryptionconfig.GetTransformerOverrides(s.Etcd.EncryptionProviderConfigFilepath)
```

Comment:
I think its better to leave the config management to the cloud-provider. Especially when we get to things like hybrid kubernetes. I agree it would be good to be consistent with the structure we feed the configuration into, but that doesn't mean we need to have opinions on the the config file format or location.